src/util/redux.spec.js

```diff
@@ -0,0 +1,197 @@
+import assert from 'assert';
+import sinon from 'sinon';
+import {
+	assign,
+	isFunction,
+	upperCase
+} from 'lodash';
+
+import {
+	thunk,
+	getReduxPrimitives
+} from './redux';
+
+describe('getReduxPrimitives', () => {
```

Comment:
we should group all these in a `describe('redux utils')` a module-level grouping.